docs/developer/extending/apps/architecture/app-permissions.mdx

```diff
@@ -0,0 +1,87 @@
+---
+title: App permissions
+---
+
+[Permissions](developer/permissions.mdx) are a way to control what operations can be performed by an app.
+
+## Managing permissions
+
+When you install an app, Saleor grants it a set of permissions. The initial app permissions are drawn from [the app's manifest file](developer/extending/apps/architecture/manifest.mdx) and stored in the [`App`](api-reference/apps/objects/app.mdx) object under the `permissions` field.
+
+Permissions can also be modified after the app is installed. You can change them using the [appUpdate](api-reference/apps/mutations/app-update.mdx) mutation:
```

Comment:
`MANAGE_APPS` permission is required for this operation

Reply:
added